src/Infrastructure/Persistence/Doctrine/Migrations/Version20240429123421.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Persistence\Doctrine\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20240429123421 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE named_street ALTER city_code SET NOT NULL');
```

Comment:
Peut-on intégrer ces SET NOT NULL à la migration précédente ?

Reply:
Elle a déjà été exécutée sur mon environnement et je me demande si ce n'est pas aussi le cas en staging (il ne doit pas reconstruire l'entièreté de la DB à chaque déploiement). Perso ca ne me dérange pas de les laisser en séparer.

Reply:
> il ne doit pas reconstruire l'entièreté de la DB à chaque déploiement

Si je pense qu'il crée une nouvelle DB (elle est vide) lors de la création de l'app de revierw, puis y applique les migrations quand on déploie

Mais du coup vu que ça a déjà été déployé, l'app de review a déjà cette migration en effet

Après ça n'est pas forcément incompatible, on peut regrouper les migrations et ça sera ok lors du déploiement en prod

Mais bon par simplicité ok pour laisser comme ça